src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,119 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int INITIAL_SRC_POS = 0;
+    private static final double GROW_SIZE = 1.5;
+    private T[] arrayData;
+    private int size;
+    private int fullSizeOfArray = 10;
```

Comment:
let's make this variable constant field and rename it to INITIAL_CAPASITY or smth like that